lib/components/Radio/Radio.test.tsx

```diff
@@ -4,6 +4,10 @@ import { render } from '@testing-library/react';
 import { BraidTestProvider, Radio } from '..';
 
 describe('Radio', () => {
+  beforeAll(() => {
+    jest.spyOn(console, 'warn').mockImplementation();
+  });
```

Comment:
Mocking console warn for component tests with deprecation messages to clean up test output.